### PR TITLE
(docs) Updating Sentry: 3rd Party Middlewares

### PR DESCRIPTION
**Pull request by smeubank**
Sentry has a dedicated integration for Starlette. Checking the other 3rd part support here, I thought it might be relevant to add the support for performance data collection, including links to the OpenTelemetry support

Discussion comment: https://github.com/encode/starlette/discussions/1787#discussioncomment-3314878

Related PR: https://github.com/encode/starlette/pull/2044

Apologies, I am a bit late to following up from the discussion :)

### CODE REVIEW

**Comment by smeubank** [comment]
checking the list I saw something related to profiling. Sentry has profiling support for [Starlette in beta](https://docs.sentry.io/platforms/python/guides/starlette/profiling/). Would it also be interesting to include that here?

File: docs/middleware.md [hunk 1]
```diff
@@ -773,9 +773,9 @@ A middleware class for reading/generating request IDs and attaching them to appl
 
 A middleware class for logging exceptions, errors, and log messages to [Rollbar](https://www.rollbar.com).
 
-#### [SentryMiddleware](https://github.com/encode/sentry-asgi)
+#### [SentryMiddleware](https://docs.sentry.io/platforms/python/guides/starlette/)
 
-A middleware class for logging exceptions to [Sentry](https://sentry.io/).
+Sentry provides a dedicated integration for Starlette. A middleware class for logging exceptions to [Sentry](https://sentry.io/). Integrations available for collecting performance data, in the form of spans and traces. If you already use [OpenTelemetry](https://docs.sentry.io/platforms/python/guides/starlette/performance/instrumentation/opentelemetry/), there is also support to send those traces and spans with their SDK.
```
Comment (Kludex) [inline docs/middleware.md:778]:
```suggestion
A middleware class for logging exceptions to [Sentry](https://sentry.io/). Integrations available for collecting performance data, in the form of spans and traces. If you already use [OpenTelemetry](https://docs.sentry.io/platforms/python/guides/starlette/performance/instrumentation/opentelemetry/), there is also support to send those traces and spans with their SDK.
```

**Review by Kludex — COMMENTED** [review]
I guess it needs a rebase :eyes:

**Comment by Kludex** [comment]
Actually, the Sentry integration doesn't expose the `SentryMiddleware` class. I guess it doesn't make sense to include this PR.

I'm going to open a PR removing the current `SentryMiddleware` entry on that part of the docs.

I guess we want to include this either on `Third Party Packages` or on the `Exceptions`.

Ref.: https://docs.sentry.io/platforms/python/guides/starlette/

**Comment by Kludex** [comment]
We already removed it on https://github.com/encode/starlette/pull/2044.

**Pull request closed without merging**